node/network/approval-distribution/src/tests.rs

```diff
@@ -50,7 +51,8 @@ fn test_harness<T: Future<Output = VirtualOverseer>>(
 	let pool = sp_core::testing::TaskExecutor::new();
 	let (context, virtual_overseer) = test_helpers::make_subsystem_context(pool.clone());
 
-	let subsystem = ApprovalDistribution::new(Default::default());
+	let keysytore = test_helpers::mock::make_ferdie_keystore();
+	let subsystem = ApprovalDistribution::new(Default::default(), keysytore);
```

Comment:
Of course this is WIP. Just noticed strange var name "keysytore". Is it not by chance?

Reply:
nice catch, lol, typo. This is throw away changes mostly, just for testing purposes.